bin/sdk/gutenberg.js

```diff
@@ -2,26 +2,28 @@
 /**
  * External dependencies
  */
+const fs = require( 'fs' );
 const path = require( 'path' );
 
-exports.config = ( {
-	argv: { editorScript, viewScript, outputDir, outputEditorFile, outputViewFile },
-	getBaseConfig,
-} ) => {
+const EDITOR_FILENAME = 'editor';
+const VIEW_FILENAME = 'view';
+
+exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 	const baseConfig = getBaseConfig( {
 		cssFilename: '[name].css',
 		externalizeWordPressPackages: true,
 	} );
-	const name = path.basename( path.dirname( editorScript ).replace( /\/$/, '' ) );
+	const editorScript = path.join( inputDir, `${ EDITOR_FILENAME }.js` );
+	const viewScript = path.join( inputDir, `${ VIEW_FILENAME }.js` );
```

Comment:
Should we allow also `.jsx` files here? Some blocks might be so simple that everything is in one file.

Happy to keep it as `.js` for simplicity, though.

Reply:
I was considering it, but TBH, I'd like to keep entry point filenames as unambiguous as possible.